testutil/gulp-mocha-runner.js

```diff
@@ -44,42 +46,73 @@ var plugin = function (options) {
     }
 
     function transform(file, enc, done) {
+
         if (file.isNull()) {
             return done(null, file);
         }
         if (file.isStream()) {
             return done(new PluginError(PluginName, "Streaming not supported"));
         }
 
-        var args = [ __dirname + "/phantom-script.js", file.path ];
-        if (options.test) {
-            args.push(options.test);
-        }
-        var phantom = spawn(require("phantomjs").path, args);
-        
-        phantom.stdout.setEncoding('utf8');
-        phantom.stdout.on('data', function(data) {
-            var msgStr = data.toString().trim();
-            
-            if (msgStr.charAt(0) === "[") {
-                var msgObj;
+        (async function() {
+            async function launchChrome() {
+                return await chromeLauncher.launch({
+                    chromeFlags: [
+                        "--disable-background-timer-throttling",
+                        "--disable-default-apps",
+                        "--disable-device-discovery-notifications",
+                        "--disable-gpu",
+                        "--disable-popup-blocking",
+                        "--disable-renderer-backgrounding",
+                        "--disable-translate",
+                        "--headless",
+                        "--no-default-browser-check",
+                        "--no-first-run",
+                        "--autoplay-policy=no-user-gesture-required",
+                        "--allow-file-access-from-files"
+                    ],
+                    logLevel: "error"
+                });
+            }
+
+            const chrome = await launchChrome();
+            const protocol = await CDP({port: chrome.port});
```

Comment:
I've been experimenting with puppeteer in a different context.  It's cool to see the CDP used directly.